OTAnalytics/plugin_ui/cli.py

```diff
@@ -140,6 +140,7 @@ def _run_analysis(
         for ottrk_file in tqdm(ottrk_files, desc="Analyzed files", unit=" files"):
             save_path = self._determine_eventlist_save_path(ottrk_file)
             tracks = self._parse_tracks(ottrk_file)
+            # events = self._intersect.run_parallel(tracks, sections)
```

Comment:
Is this line necessary?

Reply:
Yes, it's not necessary. I forgot to delete that one!